Project3/style.css

```diff
@@ -0,0 +1,152 @@
+figure {
+    position: relative;
+
+
+}
+
+figure img {
+    transition: all 1s;
+    width: 200px;
+    height: auto;
+    margin: 0%;
+}
+
+figure img:hover {
+    transform: scale(1.3);
+}
+
+figcaption {
+    display: none;
+    text-align: left;
+    font-family: 'Heebo', sans-serif;
+    position: relative;
+    width: 50%;
+
+}
+
+h1 {
+    font-family: 'Heebo', sans-serif;
+    text-align: center;
+    color: rgb(255, 255, 255);
+    font-size: 100px;
+    font-display: fallback;
+}
+
+h2 {
+    font-family: 'Mukta Mahee', sans-serif;
+    text-align: left;
+    color: rgb(255, 255, 255);
+    font-size: 50px;
+}
+
+#hour1:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour2:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour3:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour4:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255);
+}
+
+#hour5:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour6:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour7:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour8:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour9:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour10:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour11:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour12:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour13:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+#hour14:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
+}
+
+@media (min-width: 375px) {
```

Comment:
Instead of writing this media query for your smallest screen size, you should write these as your base styles (no media query) and then alter them *with* media queries for later breakpoints.